DiagnosticsExtension/Models/ConnectionStringValidator/QueueStorageValidator.cs

```diff
@@ -128,26 +76,33 @@ public async Task<TestConnectionData> TestConnectionStringViaAppSetting(string a
             {
                 try
                 {
-                    value = Environment.GetEnvironmentVariable(appSettingName + "__queueServiceUri");                    
-                    appSettingClientIdValue = Environment.GetEnvironmentVariable(appSettingName + "__clientId");
-                    appSettingClientCredValue = Environment.GetEnvironmentVariable(appSettingName + "__credential");
-                    Uri objuri = new Uri(value);
-                    if (!string.IsNullOrEmpty(appSettingClientIdValue))
+                    string serviceUriString = Environment.GetEnvironmentVariable(appSettingName + "__queueServiceUri");
```

Comment:
nit: you used static/const string from the utility class in other areas but not in this class?

Reply:
fixed